backend/apid/graphql/service.go

```diff
@@ -105,7 +116,7 @@ func NewService(cfg ServiceConfig) (*Service, error) {
 	schema.RegisterHookList(svc, &hookListImpl{})
 
 	// Register handler types
-	schema.RegisterHandler(svc, &handlerImpl{factory: clientFactory})
+	schema.RegisterHandler(svc, &handlerImpl{client: cfg.MutatorClient})
```

Comment:
Is it desired to use the mutator client to register a handler?

Reply:
The handlerImpl needs to be able to reference mutators for reasons that I can't quite recall right now. If you look at its methods it does make sense.

Reply:
Okay I just wanted to make sure it wasn't a mistake!